src/main/java/seedu/address/model/person/DateOfBirth.java

```diff
@@ -50,6 +51,29 @@ public DateOfBirth(String date) {
         this.isEmpty = false;
     }
 
+    /**
+     * Returns true if DateOfBirth is empty, false otherwise.
+     * @return boolean
+     */
+    public boolean isEmpty() {
+        return this.isEmpty;
+    }
+
+    /**
+     * Takes in the {@code DateOfBirth} argument as well as a boolean to signify if "NA" can be taken as an argument
+     * to return an empty {@code DateOfBirth}, returns a newly constructed {@code DateOfBirth} with the arguments given.
+     * @param dob
+     * @param isNaAllowed
+     * @return DateOfBirth
+     */
+    public static DateOfBirth getDateOfBirth(String dob, boolean isNaAllowed) {
+        if (isNaAllowed & dob.equals("NA")) {
```

Comment:
Should probably use the constant you defined at the top of the file 😄

```suggestion
        if (isNaAllowed & dob.equals(EMPTY_DOB)) {
```